apps/nouns-camp/src/store.js

```diff
@@ -858,6 +861,7 @@ const createStore = ({ initialState, publicClient }) =>
           set((storeState) =>
             mergeSubgraphEntitiesIntoStore(storeState, {
               proposals: proposalsWithTimestamps,
+              proposalVersions: data.proposalVersions,
```

Comment:
I think you can just remove this, it should be automatic. The reason the proposals need to be manually merged is because it’s data not from the subgraph (the timestamps). All calls to `subgraphFetch` automatically updates the store.

Reply:
hm don't think it was picking them up without his, but I'll have another look

Reply:
you're right. i wonder why this failed for me earlier 🤔